src/Strings.cpp

```diff
@@ -17,6 +17,7 @@ Strings::Strings() {
     strings.insert({ "Jump", "Jump(UpArrow): " });
     strings.insert({ "Shoot", "Shoot(Spacebar): " });
     strings.insert({ "NewLine", "\n" });
+    strings.insert({ "LoadLevel", "\nTo load progress press R" });
     strings.insert({ "PressEnter", "Press enter to advance dialogue" });
     strings.insert({ "ChadRejected", "You were supposed to say yes, but I will move on" });
```

Comment:
Change this string to say "Pffft, I don't need you anyway! You made the biggest mistake of your life, and you're gonna regret turning me down!"